content/docs/guides/login-paywalls.md

```diff
@@ -0,0 +1,8 @@
+---
+$title: Login & paywall
+$order: 7
+---
+
+ Web pages may require to control the document viewing experience for logged in users, and in case of news pages, for subscribers, metered users and anonymous users. While publishing AMP pages, it's possible to hide sections of a page based on an authorization flow by using the [amp-access](https://www.ampproject.org/docs/reference/components/amp-access).
+
+ [This](https://www.ampproject.org/docs/reference/content/docs/get_started/login_requiring.md) is a tutorial on how implement a login flow with AMP.
```

Comment:
Also, I think the link for "This" is incorrect, and it should be either: 
https://www.ampproject.org/docs/get_started/login_requiring.html or  /docs/get_started/login_requiring.html